commands/filescommands/objectcommands/uploaddir.go

```diff
@@ -0,0 +1,235 @@
+package objectcommands
```

Comment:
This seems like a command that belongs in `containercommands`. Users may not be creating any objects if they just have a hierarchy of directories. Or is the reasoning that it's here because users _could_ or _will likely_ upload objects?